README.md

```diff
@@ -648,6 +650,54 @@ defineSupportCode(({ When }) => {
 })
 ```
 
+## Helpers
+
+### Cast helper
+
+Cast helper can be used to cast values for custom gherkin rules.
+To find more about casting see [Type System](#type-system).
+
+#### Cast usage
+
+This must be used on gherkin arrays. Based on your array type you have to use: 
+ * ```step.hashed()``` -> ```Cast.objects(step.hashes())```
```

Comment:
`step.hashes()`, also note that for inline code you can use a single tick instead of three 
```
* `step.hashes()` -> `Cast.objects(step.hashes())`
```